### PR TITLE
zh-CN: update Glossary/Preflight_request

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/preflight_request/index.md [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-title: Preflight request
+title: 预检请求
 slug: Glossary/Preflight_request
+l10n:
+  sourceCommit: 38fb31b12de1019a538e868a90dd4dd4858e4871
 ---
 
 {{GlossarySidebar}}
 
 一个 CORS 预检请求是用于检查服务器是否支持 {{Glossary("CORS")}} 即跨域资源共享。
```
Comment (yin1999) [inline files/zh-cn/glossary/preflight_request/index.md:10]: 请更新这句话的翻译

File: files/zh-cn/glossary/preflight_request/index.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
 
-它一般是用了以下几个 HTTP 请求首部的 {{HTTPMethod("OPTIONS")}} 请求：{{HTTPHeader("Access-Control-Request-Method")}} 和 {{HTTPHeader("Access-Control-Request-Headers")}}，以及一个 {{HTTPHeader("Origin")}} 首部。
+它一般是用了以下几个 HTTP 请求标头的 {{HTTPMethod("OPTIONS")}} 请求：{{HTTPHeader("Access-Control-Request-Method")}} 和 {{HTTPHeader("Access-Control-Request-Headers")}}，以及一个 {{HTTPHeader("Origin")}} 标头。
```
Comment (yin1999) [inline files/zh-cn/glossary/preflight_request/index.md:12]:
```suggestion
它一般是用了以下几个 HTTP 请求标头的 {{HTTPMethod("OPTIONS")}} 请求：{{HTTPHeader("Access-Control-Request-Method")}} 和 {{HTTPHeader("Access-Control-Request-Headers")}}，以及可选的 {{HTTPHeader("Origin")}} 标头。
```

File: files/zh-cn/glossary/preflight_request/index.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
 
-当有必要的时候，浏览器会自动发出一个预检请求；所以在正常情况下，前端开发者不需要自己去发这样的请求。
+当有必要的时候，浏览器会自动发出一个预检请求；所以在正常情况下，前端开发者不需要自己去发这样的请求。预检请求会在请求被标记为[“需要预检的请求”](/zh-CN/docs/Web/HTTP/CORS#preflighted_requests)时进行，而对于[简单请求](/zh-CN/docs/Web/HTTP/CORS#simple_requests)则不会进行。
```
Comment (yin1999) [inline files/zh-cn/glossary/preflight_request/index.md:14]:
```suggestion
当有必要的时候，浏览器会自动发出预检请求；所以在正常情况下，前端开发者不需要自己去发这样的请求。预检请求会在请求被标记为[“需要预检”](/zh-CN/docs/Web/HTTP/CORS#预检请求)时进行，而对于[简单请求](/zh-CN/docs/Web/HTTP/CORS#简单请求)则不会进行。
```

File: files/zh-cn/glossary/preflight_request/index.md [hunk 4]
```diff
@@ -1,28 +1,30 @@
...
 
-举个例子，一个客户端可能会在实际发送一个 `DELETE` 请求之前，先向服务器发起一个预检请求，用于询问是否可以向服务器发起一个 {{HTTPMethod("DELETE")}} 请求：
+例如，一个客户端可能会在实际发送一个 `DELETE` 请求之前，先向服务器发起一个预检请求，用于询问是否可以向服务器发起一个 {{HTTPMethod("DELETE")}} 请求：
```
Comment (yin1999) [inline files/zh-cn/glossary/preflight_request/index.md:16]:
```suggestion
例如，客户端可能会在实际发送 {{HTTPMethod("DELETE")}} 请求之前，先向服务器发起预检请求，用于询问是否可以向服务器发起 `DELETE` 请求：
```

**Review by yin1999 — COMMENTED** [review]
一些建议

File: files/zh-cn/glossary/preflight_request/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/preflight_request/index.md:10]: 没注意到lol，sorry

A CORS preflight request is a [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) request that checks to see if the CORS protocol is understood and a server is aware using specific methods and headers.

```suggestion
{{glossary("CORS")}} 预检请求是通过使用特定方法与标头检查服务器是否理解、在乎 CORS 协议的请求。
```

File: files/zh-cn/glossary/preflight_request/index.md [hunk 5]
```diff
@@ -1,28 +1,30 @@
...
+title: 预检请求
 slug: Glossary/Preflight_request
+l10n:
+  sourceCommit: 38fb31b12de1019a538e868a90dd4dd4858e4871
 ---
 
 {{GlossarySidebar}}
 
-一个 CORS 预检请求是用于检查服务器是否支持 {{Glossary("CORS")}} 即跨域资源共享。
+{{glossary("CORS")}} 预检请求是通过使用特定方法与标头检查服务器是否理解、在乎 CORS 协议的请求。
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/glossary/preflight_request/index.md:10]:
```suggestion
{{glossary("CORS")}} 预检请求用于检查服务器是否支持 CORS 协议，并且是否允许使用特定的方法和标头。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**